war/src/main/less/base/style.less

```diff
@@ -712,29 +712,19 @@ table.parameters > tbody:hover {
   display: none;
   margin-left: 20px;
   padding: 0;
+  --table-border-radius: 7px;
```

Comment:
is this some less syntax?

Reply:
Ah apologies, missed this! It isn't, it's a scoped CSS variable (so it only applies to properties using that that are inside of that selector) :)